### DISCUSSION TITLE
No recording

### DISCUSSION THREAD

**Discussion by tortho** [blakeblackshear/frigate#2954] [Ask A Question]
After lot of trial and error I finally managed to edit my old file to get recordings again.
Seem slike new options might have broken the recording.

Solved by copying config from the full config file (when I eventually found it)
https://docs.frigate.video/configuration/index#full-configuration-reference

**Comment by blakeblackshear** [comment]
Note the warning above the full config reference:

> It is not recommended to copy this full configuration file.

There are several config options specified in that config that will not be default behavior, so it is recommended to be used as a reference to build your own config.
